src/form/bf_mx.jl

```diff
@@ -210,34 +238,48 @@ function variable_mc_generation_power(pm::SDPUBFKCLMXModel; nw::Int=pm.cnw, boun
     ncnds = length(_PMs.conductor_ids(pm, nw))
 
     # calculate bounds for matrix variable
-    bound = Dict{eltype(gen_ids), Array{Real,2}}()
+    Pg_min = Dict{eltype(gen_ids), Array{Real,2}}()
+    Pg_max = Dict{eltype(gen_ids), Array{Real,2}}()
+    Qg_min = Dict{eltype(gen_ids), Array{Real,2}}()
+    Qg_max = Dict{eltype(gen_ids), Array{Real,2}}()
+
     for (id, gen) in _PMs.ref(pm, nw, :gen)
         bus = _PMs.ref(pm, nw, :bus, gen["gen_bus"])
-        vmax = bus["vmax"]
+        vmax = get(bus, "vmax", fill(Inf, 3))
```

Comment:
We probably don't want to hard-code 3 conductors in here, maybe refer to `pm.data["conductors"]`? That way it will be easier to find it later